modules/overrides/userconfigurable/api/api_test.go

```diff
@@ -62,7 +62,7 @@ func Test_UserConfigOverridesAPI_overridesHandlers(t *testing.T) {
 			name:           "GET",
 			handler:        overridesAPI.GetHandler,
 			req:            prepareRequest(tenant, "GET", nil),
-			expResp:        `{"forwarders":["my-other-forwarder"],"cost_attribution":{},"metrics_generator":{"processor":{"service_graphs":{},"span_metrics":{}}}}`,
```

Comment:
we can add a test here, set some pre-defined values in `overridesAPI` and extend the postJSON payload to include  different values. I was particularly interested on whether setting an empty list of host identifiers and an empty metric name would overrides previous values. Seems like it does but having test coverage would be nice. (I was interested in that particular case because it would allow us to disable host info metric generation when app o11y is disabled).